pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -286,5 +289,18 @@ func gitInfo(tr *v1beta1.TaskRun) (commit string, url string) {
 			url = r.Value
 		}
 	}
+
+	url = spdxGit(url, "")
 	return
 }
+
+// supports the SPDX format which is recommended by in-toto
+// ref: https://spdx.dev/spdx-specification-21-web-version/
+// ref: https://github.com/in-toto/attestation/blob/849867bee97e33678f61cc6bd5da293097f84c25/spec/field_types.md
+func spdxGit(url, revision string) string {
```

Comment:
I think revision can be a sha1, is that something we should test?
![image](https://user-images.githubusercontent.com/1304826/146067935-63d5d365-eba0-4968-aa86-585fbc532ba5.png)

Reply:
although that does make the "digest" a bit redundant.

Reply:
ya so if the revision is a commit ID then I just defaulted to putting it in the "digest" section instead of the URI -- does that make sense?